proxy_mobile/react-native/ReactNativeApp.js

```diff
@@ -19,7 +19,7 @@ export default class ReactNativeApp extends React.Component {
     const { planeDetection } = this.state;
     return (
       <View style={styles.container}>
-        <ARView style={styles.arView} planeDetection={planeDetection} rendersContinuously={true} />
+        <ARView style={styles.arView} planeDetection={planeDetection} rendersContinuously={true} showLayoutBounds={false}/>
```

Comment:
The attribute `showLayoutBounds` shows/hides bounds for all nodes/layers, doesn't it?
In iOS, the user can set `debug` mode per node/layer (I guess iOS `debug` is equivalent to Android `showLayoutBounds`???).
Showing layout bounds per node gives the user more control over analyzing the scene.

Reply:
I agree in 50%, I think the best idea is to give developer the ability to use "debug" mode globally and per component